README.md

```diff
@@ -33,20 +33,19 @@ Unlike most other ECS libraries out there, it provides
 ## Example
 
 ```rust
+#[macro_use]
+extern crate specs;
```

Comment:
Let's not have any `extern crate`s or imports in the README, it's irrelevant for the API usage.